pandas/core/indexes/datetimelike.py

```diff
@@ -398,15 +398,17 @@ def _convert_scalar_indexer(self, key, kind=None):
 
         assert kind in ["loc", "getitem", "iloc", None]
 
+        if not is_scalar(key):
+            raise TypeError(key)
```

Comment:
what is this now caught by?

Reply:
The check in indexing L1800 prevents us from actually getting here

Reply:
kk